README.md

```diff
@@ -2,13 +2,27 @@
 
 This repository contains shared code used by targeting, frontend, and MAPI to create and process targeted campaigns.
 
-# Version mapping
-
-|Targeting Client Version|Play Version|Scala Version     |Artefact                                            |
-|------------------------|------------|------------------|----------------------------------------------------|
-|0.14.8                  |2.6         |2.11 & 2.12       |`"com.gu" %% "targeting-client-play26" % "0.14.8"`  |
-|1.0.x                   |2.7         |2.11 & 2.12 & 2.13|`"com.gu" %% "targeting-client" % "1.0.0"`          |
-|1.1.x                   |2.8         |2.12 & 2.13       |`"com.gu" %% "targeting-client" % "1.1.0"`          |
+Versions
+--------
+
+### Supported Play Versions
+
+* Play **3.0** : use [![targeting-client artifacts](https://index.scala-lang.org/guardian/grid/targeting-client/client-play-json-v30/latest-by-scala-version.svg)](https://index.scala-lang.org/guardian/grid/targeting-client/client-play-json-v28/)
```

Comment:
I don't know why the `targeting-client` is in [`guardian/grid/artifacts/targeting-client`](https://index.scala-lang.org/guardian/grid/artifacts/targeting-client) in Scaladex. Is there a way to verify these svg urls are correct before performing a release?

Reply:
> I don't know why the `targeting-client` is in [`guardian/grid/artifacts/targeting-client`](https://index.scala-lang.org/guardian/grid/artifacts/targeting-client) in Scaladex.

Yep, that is weird - I don't know why either. https://index.scala-lang.org/guardian/targeting-client is 404.



> Is there a way to verify these svg urls are correct before performing a release?

I think lets do a release first, and then work out the urls after... but note that [we've currently got the release workflow file in the wrong place](https://github.com/guardian/targeting-client/pull/37/files#r1434970608)!

Reply:
Good shout! Raised [this](https://github.com/guardian/targeting-client/pull/39) so that we can trigger the workflow.